tidevice/__main__.py

```diff
@@ -323,8 +323,16 @@ def cmd_relay(args: argparse.Namespace):
 def cmd_wdaproxy(args: argparse.Namespace):
     """ start xctest and relay """
     d = _udid2device(args.udid)
+    
+    env = {}
+    for kv in args.env or {}:
```

Comment:
args.env is list not dic

Reply:
Also, the code looks correct to me. Args is the dict of the CLI arguments. args.env is a list as you say since there can be multiple -env arguments. Each one is a "key:val" as documented. This converts that into a dict and passes it forwards. What's the issue?